app/src/Command/ExportListCommandsCommand.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ExportListCommandsCommand extends Command
+{
+    protected static $defaultName = 'app:list-commands';
+
+    protected function configure(): void
+    {
+        $this
+          ->setDescription('List all available Symfony commands with their documentation.')
+          ->setHelp('This command lists all available Symfony commands and their descriptions.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $commands = $this->getApplication()->all();
+        $commandList = [];
+
+        foreach ($commands as $name => $command) {
+            $description = $this->sanitizeText($command->getDescription());
+            $synopsis = $this->sanitizeText($command->getSynopsis());
+            $helper = $this->sanitizeText($command->getHelp());
+
+            $commandList[$name]['description'] = $description;
```

Comment:
From a coding style point of view don't use temporary variable, this clutter our head when we read code, needing to remember declared code. This section is pretty small but imagine having a 100 lines of code with temporary variable at the beginning of.

What I mean by temporary variable in this case is that they are assigned and only use once.

I would write:

```
$commandList[$name] = [
  'description' => $this->sanitizeText($command->getDescription()),
  'synopsis' => $this->sanitizeText($command->getSynopsis()),
  'help' => $this->sanitizeText($command->getHelp()),
];
```